target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -1501,20 +1533,26 @@ static int rtl8367b_detect(struct rtl8366_smi *smi)
 			"chip mode");
 		return ret;
 	}
-
-	switch (chip_ver) {
-	case 0x1000:
-		chip_name = "8367RB";
-		break;
-	case 0x1010:
-		chip_name = "8367R-VB";
-		break;
-	default:
-		dev_err(smi->parent,
-			"unknown chip num:%04x ver:%04x, mode:%04x\n",
-			chip_num, chip_ver, chip_mode);
-		return -ENODEV;
-	}
+	if(chip_num == 0x6367 || chip_num == 0x0597 || chip_num == 0x0276) {
+		chip_name = "8367C";
+	} else
```

Comment:
please add curly braces or remove newline after "else" and remove one level of indentation to make it more readable